core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -145,20 +146,14 @@ export class ExplorationHtmlFormatterService {
     // set as attribute keys (like '[' or ']'). So when interaction is migrated
     // we first test whether the other parts of the attribute can be added
     // (code above) and then we add the attribute using string concatenation.
-    if (
-      this.migratedInteractions.indexOf(interactionId) >= 0 &&
-      (lastAnswerPropValue !== null || savedSolution === 'savedMemento()')
-    ) {
-      let interactionHtml = element.outerHTML;
-      const tagEnd = '></oppia-interactive-' + htmlInteractionId + '>';
-      let interactionHtmlWithoutEnd = interactionHtml.replace(tagEnd, '');
-      if (savedSolution === 'savedMemento()') {
-        interactionHtmlWithoutEnd += ` [saved-solution]="${savedSolution}"`;
-      }
-      interactionHtmlWithoutEnd += ` [last-answer]="${lastAnswerPropValue}"`;
-      return interactionHtmlWithoutEnd + tagEnd;
+    let interactionHtml = element.outerHTML;
+    const tagEnd = '></oppia-interactive-' + htmlInteractionId + '>';
+    let interactionHtmlWithoutEnd = interactionHtml.replace(tagEnd, '');
+    if (savedSolution === 'savedMemento()') {
+      interactionHtmlWithoutEnd += ` [saved-solution]="${savedSolution}"`;
```

Comment:
Can you explain why we no longer need these changes? Apart from the fact that this place is not tested?

Reply:
see, this function return Interactions into its Html, and the part i change is end of every interaction(after last answer prop). It is either end exploration or graph interaction( connect with other graph interaction). Now both are migrated into angular. So we need to set attributes by string concatenation.

so basically now for both graph and end exploration, we remove Attribute using 

```ruby
if (this.migratedInteractions.indexOf(interactionId) >= 0) {
      element.removeAttribute('last-answer');
    }
```
in line 139.
and set attributes with (basically '[' ']') string concatenation.

for better idea we should wait for all interactions to be migarted in code base. So my thinking is to hold this part for now.

Reply:
I suggest holding this for now, we can remove it once all interactions are migrated.

Reply:
Ya sure.

Reply:
I agree with @ashutoshc8101

Reply:
Done